### PR TITLE
Bump engine version

### PR DESCRIPTION
**Pull request by roblourens**

### CODE REVIEW

**Review by daviddossett — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
         "theme": "light"
     },
     "engines": {
-        "vscode": "^1.74.0-insider"
+        "vscode": "^1.75.0"
```
Comment (DonJayamanne) [inline package.json:32]: Shouldn't this be 1.75.0-insider

**Review by DonJayamanne — CHANGES_REQUESTED** [review]
I think the engine version is incorrect

File: package.json (hunk 1 above)
Comment (roblourens) [reply package.json:32]: No, we decided in Slack that `-insider` doesn't do anything.

Comment (DonJayamanne) [reply package.json:32]: Oh it doesn't, ok, i ididn't know that.

**Pull request merged**